tests/commands/create/test_install_app_requirements.py

```diff
@@ -491,6 +674,130 @@ def test_app_requirements_requires(
     assert myapp.test_requires is None
 
 
+def test_app_requirements_requirement_installer_args_no_template_support(
+    create_command,
+    myapp,
+    app_path,
+    app_requirements_path,
+    mock_now,
+    app_requirements_path_index,
+):
+    """If an app has requirement install args, a requirements file is still written,
+    but no requirement installer args file is written if the template does not support it.
+    """
+    myapp.requirement_installer_args = ["--no-cache"]
+    myapp.requires = ["my-favourite-package"]
+
+    # Install requirements into the bundle
+    create_command.install_app_requirements(myapp, test_mode=False)
+
+    # requirements.txt exists either
+    assert app_requirements_path.exists()
+    with app_requirements_path.open(encoding="utf-8") as f:
+        assert f.read() == f"{GENERATED_DATETIME}\nmy-favourite-package\n"
+
+    # Original app definitions haven't changed
+    assert myapp.requires == ["my-favourite-package"]
+    assert myapp.test_requires is None
+
+
+def test_app_requirements_requirement_installer_args_with_template_support(
+    create_command,
+    myapp,
+    app_path,
+    app_requirements_path,
+    app_requirement_installer_args_path,
+    mock_now,
+    app_requirement_installer_args_path_index,
+):
+    """If an app has requirement install args, a requirements file is still written,
+    and requirement installer args file is written if the template supports it."""
+    myapp.requirement_installer_args = ["--no-cache", "-f", "wheels with space"]
+    myapp.requires = ["my-favourite-package"]
+
+    # Install requirements into the bundle
+    create_command.install_app_requirements(myapp, test_mode=False)
+
+    # requirements.txt exists either
+    assert app_requirements_path.exists()
+    with app_requirements_path.open(encoding="utf-8") as f:
+        assert f.read() == f"{GENERATED_DATETIME}\nmy-favourite-package\n"
+
+    assert app_requirement_installer_args_path.exists()
+    assert (
+        app_requirement_installer_args_path.read_text(encoding="utf-8")
+        == "--no-cache\n-f\nwheels with space"
```

Comment:
A mostly cosmetic thing - but should this have a trailing newline at the end?

Reply:
I'd left out the trailing newline because I was worried about needing to filter empty lines in the templates, but the templates already do anyway. I can add it for consistency.

Reply:
Done in https://github.com/beeware/briefcase/pull/2059/commits/7666f2f208469d63df78c6a6bb52aac56d187aca